site/pages/api/create-disruption.test.ts

```diff
@@ -367,6 +367,338 @@ describe("create-disruption API", () => {
         expect(writeHeadMock).toBeCalledWith(302, { Location: `/create-disruption/${defaultDisruptionId}` });
     });
 
+    it("should redirect to /create-disruption when publish start time is not numeric", async () => {
+        const disruptionData = {
+            ...defaultDisruptionData,
+            publishStartTime: "uuuu",
+            disruptionStartDate: getFutureDateAsString(40),
+            disruptionStartTime: "1200",
+            validity1: [
+                defaultDisruptionStartDate,
+                "1000",
+                defaultDisruptionStartDate,
+                "1100",
+                "",
+                "daily",
+                getFutureDateAsString(11),
+            ],
+            validity2: [
+                getFutureDateAsString(11),
+                "0900",
+                getFutureDateAsString(13),
+                "1100",
+                "",
+                "weekly",
+                getFutureDateAsString(40),
+            ],
+        };
+        const { req, res } = getMockRequestAndResponse({ body: disruptionData, mockWriteHeadFn: writeHeadMock });
+
+        await createDisruption(req, res);
+
+        const errors: ErrorInfo[] = [{ errorMessage: "Enter publication start time", id: "publishStartTime" }];
```

Comment:
Similar to comment above, this error message doesn't make complete sense for an invalid inout. If we change/ add a custom error message we should update its usages within the tests.

Reply:
done